tests/plugins/org.polarsys.capella.test.diagram.tools.ju/META-INF/MANIFEST.MF

```diff
@@ -34,4 +35,4 @@ Export-Package: org.polarsys.capella.test.diagram.tools.ju,
  org.polarsys.capella.test.diagram.tools.ju.xcbd,
  org.polarsys.capella.test.diagram.tools.ju.xdfb,
  org.polarsys.capella.test.diagram.tools.ju.xfbd
-Bundle-Localization: plugin
+Bundle-Localization: plugin
```

Comment:
can you revert this line, blundle-localization is no longer working:

https://ci.eclipse.org/capella/job/capella-product/job/PR-265/1/testReport/junit/org.polarsys.capella.test.platform.ju.testcases/AboutInfoTest/AboutInfoTest/

Reply:
Ok.